inginious/frontend/pages/course_admin/api_auth.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for
+# more information about the licensing of this file.
+
+from flask import request, Response
+import jwt
+from datetime import datetime
+import json
+from bson.json_util import dumps
+
+from inginious.frontend.pages.course_admin.utils import INGIniousAdminPage
+
+
+class CourseAPIAuthPage(INGIniousAdminPage):
```

Comment:
`INGIniousPage`  should be enough. The best option is to recycle `INGIniousAPI` page and harmonize with existing code.